config/routes.rb

```diff
@@ -43,6 +43,10 @@
       match 'assignment_override', to: '/api/v1/cors#allow', via: :options
       resource :assignment_override, only: :create
 
+      # Shared secret-based batch assignment override for chrome extension
+      match 'batch_assignment_override', to: '/api/v1/cors#allow', via: :options
+      resource :batch_assignment_override, only: :create
```

Comment:
i'm open to suggestions, but i opted to drop this right into the /v1 namespace alongside the existing endpoint because it's purely additive.